pocs/camera/fli.py

```diff
@@ -69,21 +72,23 @@ def __init__(self,
 
             # Search in cached device node scanning results for serial number.
             try:
-                device_node = Camera._fli_nodes[self.port]
+                device_node = Camera._fli_nodes[kwargs['serial_number']]
             except KeyError:
-                message = 'Could not find {} ({})!'.format(self.name, self.port)
+                message = 'Could not find {} ({})!'.format(name, kwargs['serial_number'])
                 self.logger.error(message)
                 warn(message)
                 return
-            self.logger.debug('Found {} ({}) on {}'.format(self.name, self.port, device_node))
-            self.port = device_node
+            self.logger.debug('Found {} ({}) on {}'.format(
+                name, kwargs['serial_numner'], device_node))
```

Comment:
`serial_numner` -> `serial_number`

Reply:
Oops! This is the sort of thing that slips though when you're modifying hardware dependent code without the hardware on hand to test it.